### ISSUE TITLE
Users should see indication when Quiet is not connected

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#1706]

**Comment by holmesworcester** [comment]
@rajdip-b do you think this issue is a good fit? I think it boils down to:

1. Learning from libp2p when we are connected to peers
2. Logic for when to show this alert (e.g. not when there is just one peer in a fresh community)
3. Telling the the frontend to show the alert
4. Showing the alert

**Comment by rajdip-b** [comment]
Yeah totally! I think I'll pick this one up

**Comment by rajdip-b** [comment]
@holmesworcester hey, okay so finally got the time to get started with this issue. I checked the design, it refers to the mobile UI. Do we have a similar design for desktop UI aswell? If so, I would like to attempt this with desktop before i jump into the mobile UI

**Comment by holmesworcester** [comment]
![Screenshot_20240123-072354.png](https://github.com/TryQuiet/quiet/assets/213678/cbbfaf52-cb01-4f4f-b5f5-38d3b4f31380)

If you scroll over there is a design for desktop. Note that we just want to show the "trying to connect..." message for this issue and we do not need to worry about presence as that is a separate issue.

**Comment by holmesworcester** [comment]
And thanks!!! 

Oh, also the loom Jason left might be helpful!

**Comment by rajdip-b** [comment]
Ah, alright. imma check that video out then. Also, need to dig deep into how the sockets work 👀

**Comment by holmesworcester** [comment]
Yes! This issue is a deeper dive for sure. Feel free to put together lists of questions for us, as you go!

**Comment by holmesworcester** [comment]
@rajdip-b we're onboarding a new full-time person now (@ikoenigsknecht) and it might make sense for her to do this. 

We're also trying to improve documentation so perhaps you could work on it together, and @ikoenigsknecht could build up the feature and the documentation to the point where it was straightforward for you to continue. 

What do you think? Would you have time to work on this later this week or next?

**Comment by rajdip-b** [comment]
> @rajdip-b we're onboarding a new full-time person now (@ikoenigsknecht) and it might make sense for her to do this.
> 
> We're also trying to improve documentation so perhaps you could work on it together, and @ikoenigsknecht could build up the feature and the documentation to the point where it was straightforward for you to continue.
> 
> What do you think? Would you have time to work on this later this week or next?

Hey man, sorry for this delay. I am completely occupied with a project of mine and don't really feel I would be free any time soon. Do assign the issue to her, and I will ping you back to pick up issues when I'm free again!

**Comment by holmesworcester** [comment]
No problem! We decided on another issue for onboarding so this one will remain available to work on.

**Comment by holmesworcester** [comment]
Actually we changed our minds :) Isla will start this!
